packages/docs/operations-manual/attestation-service.md

```diff
@@ -34,7 +34,11 @@ Attestation Service exposes a HTTP endpoint, but it is strongly recommended that
 
 An Attestation Service is usually deployed alongside a Celo full node instance, which needs to have the attestation signer key unlocked. This can be either deployed on the same physical machine, or in a VM or container on a different host. It is possible but not recommended to use a proxy node as the associated full node, but in this case ensure RPC access is locked down only to the Attestation Service.
 
-Attestation Service currently does not support setups where multiple instances reside behind a load balancer. This will be addressed in a future release.
+Attestation Service is a stateless service that uses a database to persist status of current and recently completed SMS delivery attempts. The most straightforward deployment architecture is to have a single machine or VM running three containers: one the attestation service, a Celo Blockchain node, and a single database instance.
+
+For a high availability setup, multiple instances can be deployed behind a load balancer configured with a round robin routing policy using the instances' `/healthz` endpoint as a healthcheck. Deploying a high availability database setup is beyond the scope of these instructions, but is straightforward with most cloud providers.  In this setup, if a delivery report for an SMS issued by one instance is received by another instance, that instance can act on it and resend if necessary.
```

Comment:
Maybe its inferred, but it could be beneficial to explicitly say that all attestation service instances should share a single database instance/setup.

Reply:
edited to clarify this